apps/ashop/src/helpers/cruiseInfo.ts

```diff
@@ -1,13 +1,14 @@
 import { pouchService } from '@boatnet/bn-pouch';
 import { AshopCruise, CouchID } from '@boatnet/bn-models';
 import { remove } from 'lodash';
+import { allDocs } from './queries';
 
 let cruise: AshopCruise;
 const db = pouchService.db;
 
 export async function getCruise(type: string, docs: any) {
-    cruise = docs.filter((row: any) => row.doc.type === type + '-cruise');
-    cruise = cruise[0] ? cruise[0].doc : undefined;
+    cruise = docs.filter((row: any) => row.type === type + '-cruise');
```

Comment:
This cool, but I think you might want to create a view in Pouch that does this. 
Parsing allDocs may be more of a perf hit that we want.

Reply:
good point, i'll create an issue for this and work on it next: https://github.com/nwfsc-fram/boatnet/issues/1193

Gonna just push this code in for now, trying to get a-shop in prod working and there's some changes in here i want to get out.